src/diffusers/pipelines/wan/pipeline_wan_i2v.py

```diff
@@ -599,7 +616,8 @@ def __call__(
         if negative_prompt_embeds is not None:
             negative_prompt_embeds = negative_prompt_embeds.to(transformer_dtype)
 
-        image_embeds = self.encode_image(image, device)
+        if not image_embeds:
```

Comment:
```suggestion
        if image_embeds is None:
```

We can't do `not <tensor>`: `Boolean value of Tensor with more than one value is ambiguous`

```python
import torch
from typing import Optional

image_embeds: Optional[torch.Tensor] = None

if not image_embeds:
    print("encode_image")

image_embeds: Optional[torch.Tensor] = torch.randn([1, 1024])

if not image_embeds:
    print("encode_image")
```

cc @yiyixuxu for awareness